crates/xlineutl/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "xlineutl"
+version = "0.1.0"
+edition = "2021"
+authors = ["DatenLord <dev@datenlord.io>"]
+repository = "https://github.com/xline-kv/Xline/tree/master/xlinectl"
+description = "Command line client for Xline"
+license = "Apache-2.0"
+readme = "README.md"
+categories = ["Client"]
+keywords = ["Client", "CommandLine"]
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+clap = { version = "4.3.2", features = ["derive", "string"] }
```

Comment:
Why do we need these two features here?

Reply:
Just copy from `xlinectl`, removed for both of them